src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,190 @@
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    verifyResponse(smallAngryPlanet, secondServicePointId, items);
+  }
+
+  private void createRequest(InventoryItemResource smallAngryPlanet, IndividualResource steve, UUID secondServicePointId)
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    RequestBuilder secondRequestBuilderOnItem = new RequestBuilder()
+      .open()
+      .hold()
+      .withPickupServicePointId(secondServicePointId)
+      .forItem(smallAngryPlanet)
+      .by(steve);
+    requestsClient.create(secondRequestBuilderOnItem);
+  }
+
+  private void verifyResponse(InventoryItemResource smallAngryPlanet,
+                              UUID secondServicePointId,
+                              List<JsonObject> items) {
+    JsonObject itemJson = items.get(0);
+    assertThat(itemJson.getString(BARCODE_KEY), is(smallAngryPlanet.getBarcode()));
+    assertThat(itemJson.getJsonObject(STATUS_KEY).getMap().get("name"),
+      is(ItemStatus.IN_TRANSIT.getValue()));
+    assertThat(itemJson.getString(DESTINATION_SERVICE_POINT), is(String.valueOf(secondServicePointId)));
+    final JsonObject smallAngryPlanetInstance = smallAngryPlanet.getInstance().getJson();
+    assertThat(itemJson.getString(TITLE), is(smallAngryPlanetInstance.getString(TITLE)));
+    final String contributors = String.valueOf(((JsonArray) smallAngryPlanetInstance
+      .getMap().get(CONTRIBUTORS)).getJsonObject(0).getMap().get("name"));
+    assertThat(itemJson.getJsonArray(CONTRIBUTORS)
+      .getJsonObject(0).getMap().get("name"), is(contributors));
+    Map<String, String> actualLocation = (Map<String, String>) itemJson.getMap().get("location");
+    assertThat(actualLocation.get(LOCATION_NAME), is("3rd Floor"));
+    assertThat(actualLocation.get(LOCATION_CODE), is("NU/JC/DL/3F"));
+    assertThat(actualLocation.get(LIBRARY), is("Djanogly Learning Resource Centre"));
+  }
+
+  @Test
+  public void reportIncludeMultipleDifferentItemsInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final InventoryItemResource nod = itemsFixture.basedUponNod();
+
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    loansFixture.checkOutByBarcode(nod);
+
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+    createRequest(nod, rebecca, secondServicePointId);
+
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+    loansFixture.checkInByBarcode(nod);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertThat(items.size(), is(2));
```

Comment:
The purpose of this test is to demonstrate that the right information is included for each item. We have had bugs in the past where information has been mixed up between records.

I think this test should check that the information included for each item is as expected.

Reply:
Let's do this in scope of https://issues.folio.org/browse/CIRC-519 story